### PR TITLE
Disable accept/decline invites if not verified

### PR DESCRIPTION
**Pull request by escobarjonatan**
image

### CODE REVIEW

File: src/components/invites/InvitesModal.tsx [hunk 1]
```diff
@@ -59,12 +83,27 @@ const InvitesModal: FunctionComponent = () => {
         <DialogContentText
           sx={ { backgroundColor: theme.colors.grey500, px: 3 } }
         >
-          <Typography variant="subtitle1">
-            You need to take action on these pending invitations.
-          </Typography>
+          <Stack
+            alignItems={ [null, null, "center"] }
+            columnGap={ 1 }
+            flexDirection={ [null, null, "row"] }
```
Comment (escobarjonatan) [inline src/components/invites/InvitesModal.tsx:89]: @scandycuz @dmkirshon Recently i found out that `flexDirection={ [null, null, "row"] }` is the same as doing `sx={{flexDirection:  [null, null, "row"] }}` 🪄

**Review by dmkirshon — CHANGES_REQUESTED** [review]
Just need to fix the verify button being visible on a verified account, otherwise LGTM

File: src/components/invites/InvitesModal.tsx [hunk 2]
```diff
@@ -59,12 +83,27 @@ const InvitesModal: FunctionComponent = () => {
...
+            <Typography variant="subtitle1">{ subtitleText }</Typography>
+            <Button
+              color="partners"
+              onClick={ handleVerifyProfile }
+              sx={ { textTransform: "none" } }
+              variant="outlined"
+              width="compact"
+            >
+              Verify profile
+            </Button>
```
[the remark this answers is not in the document]
Comment (escobarjonatan) [reply src/components/invites/InvitesModal.tsx:102]: Great catch, totally forgot to wrap it.

File: src/modules/song/api.ts [hunk 3]
```diff
@@ -118,6 +119,25 @@ export const extendedApi = api.injectEndpoints({
         }
       },
     }),
+    getSongMintPayment: build.query<GetSongMintPaymentResponse, string>({
```
Comment (escobarjonatan) [inline src/modules/song/api.ts:122]: This is not currently documented, added it in case we need it

File: src/pages/home/owners/Songs.tsx [hunk 4]
```diff
@@ -132,22 +132,6 @@ const Songs: FunctionComponent = () => {
                   ) }
                 </IconButton>
               ) }
-              <Stack
```
Comment (escobarjonatan) [inline src/pages/home/owners/Songs.tsx:135]: Talked to product/design, they do not think we should include the price on the song of the artist page for MVP

**Review by dmkirshon — APPROVED** [review]
LGTM

**Pull request merged**